doc/release/2.5.1.rst

```diff
@@ -44,17 +44,18 @@ Functionality added or changed
 Core
 ~~~~
 
--   **[Breaking change]** ``box.session.push()`` parameter ``sync`` is
-    deprecated and deleted. It does not work anymore, and a usage attempt
-    leads to an error (gh-4689).
+-   **[Breaking change]** :doc:`</reference/reference_lua/box_session/push>`
```

Comment:
```suggestion
-   **[Breaking change]** The ``sync`` parameter of the :doc:`</reference/reference_lua/box_session/push>` function is depricated and deleted.
```

Reply:
Not sure about that: if we are going to proof-read and correct the RN items for the previous release in our documentation.